app/views/add-config-volume.html

```diff
@@ -22,18 +22,7 @@
                 </div>
 
                 <div ng-if="targetObject && configMaps && secrets">
-                  <div ng-if="!configMaps.length && !secrets.length" class="empty-state-message empty-state-full-page">
```

Comment:
Should we keep this for the case where you aren't authorized to create secrets / config maps?